wcfsetup/install/files/lib/system/bbcode/AttachmentBBCode.class.php

```diff
@@ -55,162 +62,156 @@ public function getParsedTag(array $openingTag, $content, array $closingTag, BBC
             }
         }
 
-        /** @var Attachment $attachment */
-        $attachment = MessageEmbeddedObjectManager::getInstance()->getObject(
-            'com.woltlab.wcf.attachment',
-            $attachmentID
-        );
-        if ($attachment === null) {
-            if (self::$attachmentList !== null) {
-                $attachments = self::$attachmentList->getGroupedObjects(self::$objectID);
-                if (isset($attachments[$attachmentID])) {
-                    $attachment = $attachments[$attachmentID];
+        if ($thumbnail === false) {
+            $class = '';
+            if ($alignment === 'left' || $alignment === 'right') {
+                $class = 'messageFloatObject' . \ucfirst($alignment);
+            }
```

Comment:
This might be more readable as `$class = match ($alignment)` with explicit strings, instead of `ucfirst()`.